src/main/java/be/glever/antplus/FluxProvider.java

```diff
@@ -0,0 +1,8 @@
+package be.glever.antplus;
+
+import be.glever.ant.message.AntMessage;
+import reactor.core.publisher.Flux;
+
+public interface FluxProvider {
```

Comment:
Good idea, but 2 thoughts on this:
We could just assign the interface to the abstract AntChannel, or just add it as an abstract method.
In any case the name is not really ok. 'FluxProvider' emphasises the technology instead of intent (returning stream of events)

Reply:
Yes, I'm flexible on how to implement this.
I just quickly needed it for my application, so I just made it work :D

Do you think an interface is even needed? Or should we just add this method plus the backing field to `AntChannel`?
I'm not familiar with Flux and I haven't thought a lot about this, so I don't know what makes the most sense.

Reply:
No, I'll pull it up to the AntChannel.
I'm in the same boat with regards to Flux. My initial implementation was a home-brewn messagedispatcher, which broke down under its' own complexity.
I tried spring reactor (already did a bit of rxjs before) and it proved an excellent fit. But using Flux / Mono / etc are just a means to an  end (passing around streams of events).